exercises/dolica_akelloegwel/ex01_basics/src/main.cpp

```diff
@@ -105,13 +93,25 @@ std::map<std::string, int> CountWords(std::string inputFilename)
 		std::cout << "Error: Unable to open the file " << inputFilename << " for reading." << std::endl;
 		exit(1);
 	}
+	
+	// Declare a map for storing the word counts
+	std::map<std::string, int> wordMap;
+
+	// Remove punctuation characters
+	std::regex unwantedChars("[.,?'\"!():;]");
 
 	// Traverse through the file
 	while (!inFile.eof())
 	{
 		// Store the word in a string
 		inFile >> words;
 
+		// Convert the word(s) to lower case
+		for (unsigned int i = 0; i < words.length(); ++i)

```

Comment:
this seems like a good candidate for `std::transform` see str_tolower [here](https://en.cppreference.com/w/cpp/string/byte/tolower)